docs/start.md

```diff
@@ -56,6 +56,13 @@ OPTIONS
 
       The health check process handles 'repeat' and custom 'openPort' services.
 
+  --noPassword
+      When provided, Bootstrap will not use a password, so private keys will be stored in plain text. Use with caution.
+
+  --password=password
```

Comment:
Its a security risk to take the password on the command line.
if a user node gets hacked, the hacker can look through the shell history to find the user password.
would suggest add another parameter to give the user a more secure option.

      --password-stdin    Take the password from stdin

Reply:
Yes. Try it out. If password is not provided, bootstrap will ask for it

Reply:
I've just tried pipe | commands like the example from docker and it works really nice.
````
echo "$MY_ENV_VAR_PASSWORD" | symbol-bootstrap start -p testnet -a dual
````
````
echo "$MY_ENV_VAR_PASSWORD" | symbol-bootstrap decrypt --source target/addresses.yml --destination plain-addresses.yml
````
Added examples

Reply:
cool.  Sounds good.

Reply:
I'll merge after tomorrow's supernode release.